terraform/deployments/search-api-v2/datastore_schema.tf

```diff
@@ -0,0 +1,203 @@
+        # https://cloud.google.com/generative-ai-app-builder/docs/evaluate-search-quality)
+        cdoc_url = {
+          type        = "string"
+          retrievable = true
+        }
+        # The main title (shown in search results)
+        title = {
+          type               = "string"
+          keyPropertyMapping = "title"
+          retrievable        = true
+        }
+        # A short description (shown in search results)
+        description = {
+          type               = "string"
+          keyPropertyMapping = "description"
+          retrievable        = true
+        }
+        # Additional textual content such as keywords that should be searchable but don't form part
+        # of the main body of content
+        #
+        # Note: We've decided not to use this as structured fields don't contribute enough to
+        # relevance. Vertex does not support removing a field from the schema (as of Jan 2024) so it
+        # stays here.
+        additional_searchable_text = {
+          type       = "string"
+          searchable = true
+        }
+        # URI reference either as a relative path for GOV.UK content, or as an absolute URL for
+        # external content (used to link to the content from search results)
+        link = {
+          type        = "string"
+          retrievable = true
+          indexable   = true
+        }
+        # Absolute URL including protocol and host even for content on GOV.UK proper (used for
+        # Vertex to incorporate popularity/event signals and for internal purposes)
+        url = {
+          type               = "string"
+          retrievable        = true
+          keyPropertyMapping = "uri"
+        }
+        # Unix timestamp of when this object was last updated (for sorting/filtering/boosting)
+        public_timestamp = {
+          type        = "integer"
+          retrievable = true
+          indexable   = true
+        }
+        # RFC3339 timestamp of when this object was last updated (for boosting and displaying)
+        public_timestamp_datetime = {
+          type        = "datetime"
+          retrievable = true
+          indexable   = true
+        }
+        # The source document type (for boosting)
+        document_type = {
+          type        = "string"
+          indexable   = true
+          retrievable = true
+        }
+        # The content purpose supergroup (for filtering/boosting)
+        content_purpose_supergroup = {
+          type        = "string"
+          indexable   = true
+          retrievable = true
+        }
+        # A list of GOV.UK taxon IDs that this content object is tagged to (for filtering)
+        part_of_taxonomy_tree = {
+          type = "array"
+          items = {
+            type               = "string"
+            keyPropertyMapping = "category"
+          }
+        }
+        # Whether the content is historic (for boosting; boosts don't support boolean)
+        is_historic = {
```

Comment:
I'm assuming here that `multipleOf`, `minimum` and `maximum` are also not supported by VAIS.